docs/HowToArchiveJournal.md

```diff
@@ -17,13 +17,134 @@ For more info on NuoDB database journaling, please consult the official [NuoDB d
 
 Since the `journal` has to write all commits durably to disk, the speed of the disk directly influences the commit latency.
 To achieve the best performance, NuoDB recommends placing the `journal` on the fastest disk available.
+In contract, the `archive` should be using the disk with the largest throughput.
 
 By default, the `journal` is located in a subdirectory of the `archive`.
 To achieve the best cost vs speed tradeoff, you can separate the journal from the archive.
 To achieve that set the database helm template values `database.sm.noHotCopy.journalPath.enabled` and `database.sm.hotCopy.journalPath.enabled` to `true` and configure it with the desired persistence settings.
```

Comment:
is it worth talking about different size requirements on the journal (HC vs NHC) and relation to journal hotcopy backup